pkg/db/connect.go

```diff
@@ -106,13 +103,7 @@ func normalizeDBParams(p *params.Database) {
 }
 
 func tryConnectConfig(ctx context.Context, config *pgxpool.Config, log logging.Logger) (*pgxpool.Pool, error) {
-	strategy := retry.LimitTime(maxWait,
-		retry.Exponential{
-			Initial: firstWait,
-			Factor:  waitGrowth,
-			Jitter:  true,
-		},
-	)
+	strategy := params.DatabaseRetryStrategy
 	var pool *pgxpool.Pool
 	var err error
 	for a := retry.Start(strategy, nil); a.Next(); {
```

Comment:
Here too, the actual error isn't being checked. Does the package expose the error types?